src/components/Articles.jsx

```diff
@@ -23,22 +24,24 @@ const Articles = () => {
       ) : (
         articles.map((article) => (
           <article key={article.article_id} className="">
-            <div className="flex ">
-              <p>{article.author}</p>
-              <p className="">.{article.created_at}.</p>
-              <p className="">{article.topic}</p>
-            </div>
-            <p className="">{article.title}</p>
-            <div className="">
-              <img src={article.article_img_url} alt="" />
-            </div>
-            <div className="flex justify-between">
-              <p className="flex text-blue-500">
-                <AiFillLike />
-                {article.votes}
-              </p>
-              <p className="">{article.comment_count} comments</p>
-            </div>
+            <Link to={`/pages/article/${article.article_id}`}>
+              <div className="flex ">
+                <p>{article.author}</p>
+                <p className="">.{article.created_at}.</p>
+                <p className="">{article.topic}</p>
+              </div>
+              <p className="">{article.title}</p>
+              <div className="">
+                <img src={article.article_img_url} alt="" />
+              </div>
+              <div className="flex justify-between">
+                <p className="flex text-blue-500">
+                  <AiFillLike />
+                  {article.votes}
+                </p>
+                <p className="">{article.comment_count} comments</p>
+              </div>
+            </Link>
           </article>
         ))
       )}
```

Comment:
You should have some loading state included. Preferably in it's own component so you can reuse it throughout your app. If i slow the network down at the moment i don't get any feedback to show me the page is loading